public/client.js

```diff
@@ -1,9 +1,9 @@
 $( document ).ready(function() {
-  var items = [];
-  var itemsRaw = [];
+  let  items = [];
+  let  itemsRaw = [];
   
   $.getJSON('/api/books', function(data) {
-    //var items = [];
+    //let  items = [];
```

Comment:
Any particular reason this is here?

Reply:
It was in the original boilerplate.  No idea why.  As you can tell, I did a search-and-replace on `var`, so this was picked up.  I didn't otherwise touch the client-side JS code.

I can remove it, of course.

Reply:
Oh, right. I missed that. It is not major, but if it serves no purpose, then it can only confuse.

Reply:
Removed it.